downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -62,6 +62,24 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
     }
     ```
 
+- Fixed a bug that previously resulted in variables that were initialized with non-isolated expressions being inferred to be `isolated` variables.
+
+    ```ballerina
+    int[] config = [];
+
+    // `config` is not an isolated expression, therefore,
+    // `configs` will not be inferred as an isolated variable.
+    int[][] configs = [[1, 2], config]; 
+
+    // Since `configs` is not inferred to be an `isolated` variable,
```

Comment:
Should we say about, previous behavior here?

Reply:
Updated the comment in [ce7f967](https://github.com/ballerina-platform/ballerina-dev-website/pull/6605/commits/ce7f96757d37deb3f06c98a3f22316e3d8af2c3d).